src/Controllers/UserController.js

```diff
@@ -134,6 +134,27 @@ export class UserController extends AdaptableController {
     });
   }
 
+  resendVerificationEmail(username, token) {
+    var where = {};
+    if (username) {
+      where.username = username;
+    }
+    if (token) {
+      where._email_verify_token = token;
+    }
+
+    var query = new RestQuery(this.config, Auth.master(this.config), '_User', where);
```

Comment:
could instead use `getUserIfNeeded` method here.

username is always automatically created right?